experimental/dds/tree/README.md

```diff
@@ -28,7 +57,6 @@ The order of the edits is:
 1. All acknowledged edits, in the order agreed upon by Fluid's consensus.
 2. All local edits (not acknowledged by Fluid yet), in the order they were applied.
 
-**Important: this DDS is no longer in active development, and a non-experimental, officially supported SharedTree is in active development by the Fluid team that will expand on its rich feature set.**
```

Comment:
Note for reviewers: Moved this to the top of the README and updated it to point to the new tree library.